docs/app/pages/components/sections/tree-view/tree-view-custom-node/snippets/app.snippet.html

```diff
@@ -0,0 +1,37 @@
+<tree-root #tree [nodes]="nodes" [options]="options" (loadNodeChildren)="onChildrenLoaded($event.node)">
```

Comment:
Could you break lines in this and other snippets to avoid horizontal scrolling in the documentation?